--- a/hw/xfree86/compat/log.c	2025-06-16 16:58:19.711996590 +0300
+++ b/hw/xfree86/compat/log.c	2025-06-16 17:00:54.736391571 +0300
@@ -5,6 +5,7 @@
 #include "include/os.h"
 
 #undef xf86Msg
+#undef xf86MsgVerb
 
 /*
  * this is specifically for NVidia proprietary driver: they're again lagging
@@ -26,3 +27,19 @@
     LogVMessageVerb(type, 1, format, ap);
     va_end(ap);
 }
+
+_X_EXPORT void xf86MsgVerb(MessageType type, int verb, const char *format, ...)
+    _X_ATTRIBUTE_PRINTF(3, 4);
+
+void
+xf86MsgVerb(MessageType type, int verb, const char *format, ...)
+{
+    LogMessageVerb(X_WARNING, 0, "Outdated driver still using xf86MsgVerb() !\n");
+    LogMessageVerb(X_WARNING, 0, "File a bug report to driver vendor or use a FOSS driver.\n");
+    LogMessageVerb(X_WARNING, 0, "Proprietary drivers are inherently unstable, they just can't be done right.\n");
+
+    va_list ap;
+    va_start(ap, format);
+    LogVMessageVerb(type, verb, format, ap);
+    va_end(ap);
+}
