diff --git a/hw/xfree86/compat/meson.build b/hw/xfree86/compat/meson.build
index 43a6beec7..75a2265a0 100644
--- a/hw/xfree86/compat/meson.build
+++ b/hw/xfree86/compat/meson.build
@@ -2,6 +2,7 @@ srcs_xorg_compat = [
     'clientexception.c',
     'log.c',
     'devices.c',
+    'xf86Helper.c',
 ]

 xorg_compat = static_library('xorg_compat',
--- a/dev/null	2023-03-29 23:29:11.796523582 +0300
+++ a/hw/xfree86/compat/xf86Helper.c	2025-06-16 18:07:53.967415950 +0300
@@ -0,0 +1,28 @@
+#include <dix-config.h>
+
+#include <X11/Xfuncproto.h>
+
+
+#include "xf86Priv.h"
+#include "xf86Bus.h"
+
+
+/*
+ * this is specifically for NVidia proprietary driver: they're again lagging
+ * behind a year, doing at least some minimal cleanup of their code base.
+ * All attempts to get in direct contact with them have failed.
+ */
+
+_X_EXPORT Bool xf86IsScreenPrimary(ScrnInfoPtr pScrn);
+
+Bool
+xf86IsScreenPrimary(ScrnInfoPtr pScrn)
+{
+    int i;
+
+    for (i = 0; i < pScrn->numEntities; i++) {
+        if (xf86IsEntityPrimary(i))
+            return TRUE;
+    }
+    return FALSE;
+}
